store/tools/go.mod

```
@@ -0,0 +1,31 @@
+module github.com/confio/ics23-iavl
```

```suggestion
module github.com/cosmos/cosmos-sdk/store/tools
```
im guess we would cut v1 for this after its merged? @robert-zaremba

---

yes